### PR TITLE
Add banner, make a-poem example work

### PR DESCRIPTION
**Pull request by chitty**
Add banner, it's all we're missing to get the a-poem example to work

https://github.com/Schwad/scarpe/blob/e48ef46fd7a2d1c573666c3882b3cb590ebe701d/examples/examples_that_do_not_yet_work/shoes-contrib/basic/a-poem.rb#L4

If the approach looks right, I can also add

      inscription
      ins
      para
      caption
      tagline
      subtitle
      title

### CODE REVIEW

File: lib/scarpe/para.rb [hunk 1]
```diff
@@ -12,6 +12,11 @@ class Para < Scarpe::Widget
     }.freeze
     private_constant :SIZES
 
+    def self.inherited(subclass)
```
Comment (DRBragg) [inline lib/scarpe/para.rb:15]: IIRC the `inherited` method on widget should still apply when `Para` is inherited since it's a subclass of widget

**Review by DRBragg — CHANGES_REQUESTED** [review]
One thing but I think the overall approach makes sense

File: lib/scarpe/banner.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Scarpe
+  class Banner < Para
```
Comment (Schwad) [inline lib/scarpe/banner.rb:2]: could we do something crazy like:

```ruby
# para_friends.rb
Para::SIZES.each do |size|
  a_new_class = Class.new(Object) do
    def initialize(*args, stroke: nil, **html_attributes)
      super
      @size = size
    end
  end
  self.const_set(size.to_s.capitalize, a_new_class)
end
```

**Review by Schwad — APPROVED** [review]
Looks good! @DRBragg 's thoughts make sense.  if we don't go meta we should instantiate all of them soon too :) thanks

File: lib/scarpe/para.rb (hunk 1 above)
Comment (chitty) [reply lib/scarpe/para.rb:15]: Yeah, that's what I thought too. But for some reason it doesn't work 🤷‍♂️ 
I see we're doing the same thing in `TextWidget` even when it's a subclass of widget:

https://github.com/Schwad/scarpe/blob/cdf15cf65faf6f98b8a8597b82e3e9455575d516/lib/scarpe/text_widget.rb#L4-L8

Comment (DRBragg) [reply lib/scarpe/para.rb:15]: Well that's weird 🤔

**Review by DRBragg — APPROVED** [review]

**Pull request merged**